frontend/src/HomePage/Folders.jsx

```diff
@@ -84,13 +86,19 @@ export const Folders = function Folders ({
       setCreationStatus(true);
       folderService
         .create(newFolderName, appType)
-        .then(() => {
+        .then((data) => {
           toast.success('Folder created.');
           setCreationStatus(false);
           setShowForm(false);
           setNewFolderName('');
           handleFolderChange({});
           foldersChanged();
+          posthogHelper.captureEvent('create_folder', {
```

Comment:
There is an event called create_new_folder in this file. Could you please double-check?